wolox-books/src/app/scss/equalizer.scss

```diff
@@ -0,0 +1,92 @@
+$equalizer-black: #000;

```

Comment:
This file isn't needed anymore. Check out https://www.npmjs.com/package/wolox-equalizer 😉

Reply:
Cool 👆